tests/xfstests/install.pm

```diff
@@ -33,6 +34,12 @@ sub log_create {
     assert_script_run($cmd);
 }
 
+sub collect_version {
+    my $file = shift;
+    my $cmd  = "cd /tmp/xfstests-dev; git rev-parse HEAD > $file; cd -; rpm -qa xfsprogs xfsdump btrfsprogs >> $file; uname -r >> $file";
```

Comment:
What about ```my $cmd  = "(cd /tmp/xfstests-dev; git rev-parse HEAD; cd -; rpm -qa xfsprogs xfsdump btrfsprogs; uname -r ) | tee $file";```

Reply:
shorter is better

Reply:
Thank you, modified. Since output for 'cd -' useless, I redirect it into /dev/null.